templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,7 +13,9 @@
    "cell_type": "markdown",
```

Comment:
<p>This is a bit confusing. </p><ol><li>Start with instructions on how to create the files and then where to place them. </li><li>The instructions say "the JSON file which defines each tiled region with their corresponding top-left corner FOVs". However its also followed by "In the 'fovs' list of that file, keep only FOVs you wish to define as a top-left corner for a distinct region" </li><li>The user should "name" the FOVs using Mibitracker before exporting the FOVs as it would make it easier than manually renaming the name key in the json file. </li><li>Why is it important to check the FOV parameters in the json file? </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Here's a new docstring that I think captures what you're saying:</p><p> </p><p>To identify each desired tiled region on the image, you must provide a <code>region_corners_path</code> that defines the top-left corner FOV for each of them.</p><p> </p><p>To generate these FOVs, use the Mibitracker to select the top-left FOV for each desired tiled region. Rename each FOV to the corresponding desired name of the tiled region it defines, then download the file (in JSON format). <code>region_corners_path</code> should be formatted just like a commercial instrument run file, with each top-left FOV contained in the <code>'fovs'</code> key. Place this file directly into the path specified in <code>json_tiling_dir</code>. Ensure this file is renamed <code>{tiled_region_prefix}_region_corners.json</code> (ex. if you set <code>tiled_region_prefix = test</code> in the notebook, the file should be renamed <code>test_tiled_region_corners.json</code>).</p><p> </p><p>For each top-left FOV, double check that the parameters <code>'name'</code>, <code>'centerPointMicrons'</code>, <code>'fovSizeMicrons'</code>, and the other parameters (ex. <code>'sectionId'</code>, <code> 'slideId'</code>, <code>'timingId'</code>) correspond with their values on the Mibitracker before using <code>region_corners_path</code> with the notebook. <code>'name'</code> defines the name of the tiled region; this must be correct because it is used to prefix the name of each FOV in the region to resolve ambiguity with FOV names in other tiled regions. <code>'centerPointMicrons'</code> and <code>'fovSizeMicrons'</code> are needed to position and size each FOV per tiled region; the same parameters defined in the top-left FOV will be used across all FOVs in the corresponding tiled region, so incorrect values will cause unwanted shifts and sizes for the entire region. The metadata parameters such as <code>'slideId'</code> are also copied from the top-left FOV to all other FOVs in its encompassing tiled region, so it's important that they're defined correctly for consistency purposes.</p>